django/db/models/expressions.py

```diff
@@ -915,9 +915,13 @@ def get_source_expressions(self):
         return [self.expression]
 
     def get_group_by_cols(self, alias=None):
-        expression = self.expression.copy()
-        expression.output_field = self.output_field
-        return expression.get_group_by_cols(alias=alias)
+        from django.db.models.sql.where import WhereNode
+
```

Comment:
Do we need an inner import?

Reply:
I imported it at the top of the file and got an import error

```
ImportError: cannot import name 'Count' from 'django.db.models.aggregates'
```